http/src/main/resources/init-resources/shutdown.sh

```diff
@@ -22,12 +34,8 @@ if [ -d '/mnt/disks/work/.jupyter' ] && [ "SHOULD_DELETE_JUPYTER_DIR" = "true" ]
     rm -rf /mnt/disks/work/.local || true
 fi
 
-# https://broadworkbench.atlassian.net/browse/IA-3186
-# This condition assumes Dataproc's cert directory is different from GCE's cert directory, a better condition would be
-# a dedicated flag that distinguishes gce and dataproc. But this will do for now
-# Attempt to fix https://broadworkbench.atlassian.net/browse/IA-3258
-if [ -f "/var/certs/jupyter-server.crt" ]
-then
+# TODO - Why do we need different docker compose commands?
```

Comment:
Probably because COS images need to run docker-compose as a container, where Dataproc has it natively installed.

It's interested we're not using the `cryptopants/docker-compose-gcr` image here, probably because the startup/shutdown scripts never actually need to pull anything from GCR?